src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -0,0 +1,59 @@
+import type { BtcTransactionStatus, BtcTransactionUi } from '$btc/types/btc';
+import type { BtcAddress } from '$lib/types/address';
+import type { BitcoinOutput, BitcoinTransaction } from '$lib/types/blockchain';
+import type { TransactionType } from '$lib/types/transaction';
+import { nonNullish } from '@dfinity/utils';
+
+export const mapBtcTransaction = ({
+	transaction,
+	btcAddress
+}: {
+	transaction: BitcoinTransaction;
+	btcAddress: BtcAddress;
+}): BtcTransactionUi => {
+	const type: TransactionType = transaction.inputs.some(
+		(input) => input.prev_out.addr === btcAddress
+	)
+		? 'send'
+		: 'receive';
+	let amount = 0;
```

Comment:
We generally follow a functional approach and are about to add an eslint rule that prevent the usage of `let` in favor of `const`. Can you refactor the code to amend this?

Reply:
Done.